src/types/index.ts

```diff
@@ -95,12 +102,18 @@ export type AnonymizedQueryResult = {
   summary: AnonymizationSummary;
 };
 
-export type AnonymizedResultColumn = {
-  name: string;
-  type: AnonymizedColumnType;
-};
-
-export type AnonymizedColumnType = ColumnType | 'aggregate';
+export type AnonymizedResultColumn =
+  | {
+      name: string;
+      type: TooltipColumnType;
+      valueToTooltip: TooltipFunction;
+    }
+  | {
+      name: string;
+      type: AnonymizedColumnType;
+    };
+
+export type AnonymizedColumnType = NoTooltipColumnType | 'aggregate';
```

Comment:
I don't like that tooltip information pollutes the type system.
I think it would be better to inline the types here or allow for empty `valueToTooltip` callbacks.

Reply:
> I don't like that tooltip information pollutes the type system.

I agree. Can the tooltip specific types be exported from the table component file?